fotoapparat/src/main/java/io/fotoapparat/parameter/range/ContinuousRange.java

```diff
@@ -0,0 +1,32 @@
+package io.fotoapparat.parameter.range;
+
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+
+/**
+ * Created by ychen on 5/25/2017.
```

Comment:
We do not put authors in JavaDocs, but you can leave a normal comment with your name on the top of the class if you like.

Please also add a proper JavaDoc for the class.